packages/core/src/components/DatePicker/YearPicker/YearPicker.tsx

```diff
@@ -25,9 +25,9 @@ export interface YearPickerProps {
 
 const YearPicker = ({ selectedDate, isYearBlocked, changeCurrentDate, "data-testid": dateTestId }: YearPickerProps) => {
   const selectedYear = selectedDate ? selectedDate.format(YEAR_FORMAT) : moment().format(YEAR_FORMAT);
-
+  const currentYear = moment().format(YEAR_FORMAT);
```

Comment:
Since the selected year is now the newly selected year I thought it would be confusing if the view of the years changes after each time the user changes a date.